include/fmt/compile.h

```diff
@@ -488,7 +488,11 @@ auto format_to_n(OutputIt out, size_t n, const S& fmt, Args&&... args)
 
 template <typename S, typename... Args,
           FMT_ENABLE_IF(detail::is_compiled_string<S>::value)>
-auto formatted_size(const S& fmt, const Args&... args) -> size_t {
+#if !FMT_MSC_VERSION || FMT_MSC_VERSION >= 1930
+FMT_CONSTEXPR20
+#endif
```

Comment:
Can we use FMT_CONSTEXPR20 unconditionally? It will still give a compile-time error when trying to use.

Reply:
Yes.
If we do not use ``fmt::formatted_size`` in constexpr context.